src/diff.sh

```diff
@@ -103,9 +103,10 @@ function diff_side_by_side()
   local file_2="$2"
   local interactive="$3"
   local flag="$4"
-  local columns=$(tput cols)
   local diff_cmd="diff -y --color=always --width=$columns"
+  local columns
```

Comment:
For the next time, I would suggest not inverting the order of declaration of variables, to make these large refactoring patches as imperceptible as possible. There is no execution difference, but sometimes the order has some meaning. There doesn't seem to be the case anywhere in this patch, so I don't think it justifies pushing another version :)